### PR TITLE
DRAFT: update Stable Release Policy

### PR DESCRIPTION
**Pull request by okwme**
Would be great to get input from @marbar3778 and @mmulji-ic on how to finalize this.
@faddat did a lot of work updating v2, v3 and v4 of gaia with changes to linting.
while the work is appreciated those kinds of changes to make sense to backport into a new release of those versions. EOL should only be bypassed by security updates.

### CODE REVIEW

**Comment by okwme** [comment]
should add a definition for "bug-fix" since bugs that impact operating the network don't matter wince the previous version networks are no longer in operation. More so it should be privilege escalation or anything that makes the software actually dangerous to operate on a user's machine. Maybe these should actually be applied to all previous versions since archive use will require them to be run.

**Comment by faddat** [comment]
#1510 
#1511 

So, code formatting and the like are in fact security updates.  If something were to happen, having maintained older gaias-- as well as people who maintain them, would be very important.  That is why it makes sense to backport changes like these-- at the very least as far back as cosmoshub-4 genesis (iirc that is v4)

**Review by tac0turtle — APPROVED** [review]
generally looks good to me. Id put a EOL for v5 since this is coming up in the middle of things

**Comment by yaruwangway** [comment]
if we find a bug in v6, and our testnet and and hub are both at v7, we fix the bug of v6, and spin off a new testnet to test them, right? if it is not bug, it's just improvement of code, do we still improve v6 when we are at v7 ?

**Comment by faddat** [comment]
@yaruwangway my opinion is that we should aim for consistency on these items:

* ci
* code formatting
* use (and verification) of latest cosmos-sdk

## Goal

The goal of standardizing ci, code formatting, and version bumps is to make it easy to correct issues, if their root cause is in the past.

File: RELEASING.md [hunk 1]
```diff
@@ -133,3 +133,18 @@ Checksums-Sha256:
  3895518436b74be8b042d7d0b868a60d03e1656e2556b12132be0f25bcb061ef  gaiad-4.0.0.tar.gz
 ```
 
+# Major Release Maintenance
+
+Major Release series continue to receive bug fixes (released as a Patch Release) until they reach End Of Life. Major Release series is maintained in compliance with the Stable Release Policy as described in this document. Note: not every Major Release is denoted as stable releases.
+
+Only the following major release series have a stable release status:
+
+v7 Theta is supported until v9 Epsilon. A fairly strict **bugfix-only** rule applies to pull requests that are requested to be included into a stable point-release.
```
Comment (mmulji-ic) [inline RELEASING.md]: Epsilon = 10

**Comment by faddat** [comment]
wdyt about this PR, with respect to this policy:

* https://github.com/cosmos/gaia/pull/1695

Is it suprerfluous?

**Comment by okwme** [comment]
@faddat my understanding is that this PR should support #1695 as in scope.
there are other items to include in the next 7.0.x release being tracked here: https://github.com/cosmos/gaia/issues/1669

**Pull request merged**